src/content/docs/ru/basics/rendering-modes.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Режимы Рендеринга"
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import RecipeLinks from '~/components/RecipeLinks.astro';
+
+Для отображения в Интернете код вашего проекта Astro должен быть **отрендерен** в HTML. 
+
+Страницы Astro, маршруты и конечные точки API могут быть либо [предварительно отрендерены на этапе сборки](#pre-rendered), либо [отрендерены сервером](#on-demand-rendered) при запросе пути. С [Островами Astro](/ru/concepts/islands/) вы также можете включить рендеринг на стороне клиента, если это необходимо.
```

Comment:
```suggestion
Страницы Astro, маршруты и конечные точки API могут быть либо [предварительно отрендерены на этапе сборки](#пререндеренные), либо [отрендерены сервером](#рендеринг-по-запросу) при запросе пути. С [Островами Astro](/ru/concepts/islands/) вы также можете включить рендеринг на стороне клиента, если это необходимо.
```

Reply:
Thank you! Sorry, I still sometimes confuse the correct links.